docs/api-usage/filtering.mdx

```diff
@@ -5,7 +5,7 @@ title: Filtering
 :::info
 
 The `where` filtering approach was introduced in **Saleor 3.11**. For now, it's only
-available on the `Attribute` type.
+available on the `Attribute` and `Product` type.
```

Comment:
It was also introduced for `Product` in 3.11? If not then I think we should separate it. You could make a list like:
<description>:
- `Attribute` added in 3.11
- `Product` added in 3.15